Makefile

```diff
@@ -18,15 +18,16 @@ AR := $(TOOLCHAIN_ROOT)/bin/avr-ar
 CPU := atmega1284p
 
 EXTERNAL_LIBS :=
-EXTERNALS := cb0r aes_gcm crypto-algorithms tinf
+EXTERNALS := cb0r aes_gcm crypto-algorithms tinf Monocypher
 
 COMMONFLAGS := -Werror -mmcu=$(CPU)
 LIBRARY_PREPROCESSOR_MACROS :=
 EXAMPLES_PREPROCESSOR_MACROS :=
 INCLUDE_DIRS := include
 CFLAGS := -std=gnu99 -fstack-usage -ffunction-sections -fdata-sections -fshort-enums -Wall
 ARFLAGS :=
-PREPROCESSOR_MACROS := _FIDO_INTERNAL
+CRYPTO_PREPROCESSOR_MACROS := NO_SOFTWARE_CRYPTO_AES_GCM_ENCRYPT NO_SOFTWARE_CRYPTO_ED25519_SIGN NO_SOFTWARE_CRYPTO_SHA512
```

Comment:
Really want to add this? Btw: Makefile will be removed in #22.

Reply:
I undid this change.